src/datasets/data_files.py

```diff
@@ -14,7 +14,7 @@
 from .utils.py_utils import string_to_dict
 
 
-DEFAULT_SPLIT = str(Split.TRAIN)
+SANITIZED_DEFAULT_SPLIT = str(Split.TRAIN)
```

Comment:
(nit) I think you can keep DEFAULT_SPLIT and call str() in sanitize_patterns, as sanitize_patterns already calls str() to the users requested splits.

Reply:
@lhoestq that makes sense if `DEFAULT_SPLIT` is used anywhere else than in `sanitize_patterns`, but this is not currently the case.